src/framework/standard/mod.rs

```diff
@@ -722,7 +730,9 @@ impl Framework for StandardFramework {
                     self.should_fail(&ctx, &msg, &mut args, &command.options, &group.options).await
                 {
                     if let Some(dispatch) = &self.dispatch {
-                        dispatch(&mut ctx, &msg, error).await;
+                        let command_name =
+                            command.options.names.get(0).copied().unwrap_or("<unnamed>");
```

Comment:
This case should be impossible to be `None`, but in the case it is, an error should be shown since it would mean a bug in the `#[command]` macro generation. I suggest using index syntax to both simplify the code and to assert that `command.options.names` has to have at least one element -- the command's main name.
```suggestion
                        let command_name = command.options.names[0];
```